docs/wiki/01_STUDY_PARAMETERS_SETUP/02_Clinical_Instruments/README.md

```diff
@@ -1 +1,257 @@
+Depending on the use case, either of these scripts will add the instrument to any relevant timepoints according to the contents of the test_battery table.  
+
+## 5) Populate Examiners
+Add examiners via the Examiners module, to specify all personnel likely to administer any questionnaires or tests for each study site. Edit each examiner to add certifications to administer specific instruments. This is a pre-requisite to beginning instrument data entry, as instrument data forms will not save without selecting a valid examiner.
+
+## 6) Testing and Troubleshooting Instruments
+Follow these steps to test an instrument:
+
+1. Register a DCC candidate  
+2. Create a timepoint for this candidate  
+3. Start its visit stage (date must be within test_battery-defined age range)  
+4. Enter sample data, testing each field's type and logic constraints.  
+
+A front-end user can only create candidates and visits for the site they belong to. Superusers can change a user's site (including their own) under the User Accounts module.
+
+#### showDatabaseQueries
+Toggling the following setting in config.xml can help in troubleshooting MySQL statements sent to the database. Set `<showDatabaseQueries>` to show (1) or hide (0) MySQL queries at the top of the browser.
+
+```
+<gui>
+    <showDatabaseQueries>0</showDatabaseQueries>
+</gui>
+```
+
+This feature does not work in all modules and may interfere with JavaScript running in the page, and should never be used in production instances.  
+
+## 7) Double Data Entry  
+Double Data Entry (DDE) is the practice of requiring users to manually input the same dataset twice in order to reduce the risk of error in data entry. For DDE-designated instruments, a participant's instrument data collection at a given visit cannot be finalized in LORIS unless both the first data entry form and second (double) data entry form are completed.  
+
+To enable an instrument for double data entry, in the Configuration module inside the Study panel, use the "Double data entry instruments" section to add instrument names. Be very careful when removing an instrument name from this list, regarding the impact on the integrity of the entire dataset.  
+
+Note: by default, all forms will have a Double Data Entry (DDE) duplicate form created in the back end (flag table and instrument table) when the instrument is populated for a session. However, the DDE form link does not appear in the front end and Double Data Entry completion is not enforced unless the instrument is specified as a Double Data Entry instrument via the Configuration module.  
+
+Data entry conflicts detected through between the first and second (double) data entry forms are resolved through the [Conflict Resolver](https://github.com/aces/Loris/wiki/LORIS-Modules#double-data-entry-dde-and-conflict-resolver-module)  
```

Comment:
The link section does not exist (see anchor) and this wiki section has been migrated to RTD. 
- check if this link still makes sense  sense the section has been removed
- if it does, use an internal RTD link instead.